File: code-coaches/easy/paint-costs/paint_costs.rb
def get_total_paint_costs(num_colors)
  base_price = 40.0
  paint_price = 5.0
  tax_percent = 10

  subtotal = base_price + (paint_price * num_colors)
  total = ((subtotal / tax_percent) + subtotal).ceil

  total
end

def main
  num_colors = gets.chomp.to_i
  puts get_total_paint_costs(num_colors)
end

if __FILE__ == $0
  main
end

File: code-coaches/easy/paint-costs/paint_costs.cpp
#include <iostream>
#include <cmath>

int get_total_paint_costs(int num_colors)
{
    double base_price = 40.0;
	double paint_price = 5.0;
	double tax_percent = 10.0;

	double subtotal = base_price + (paint_price * num_colors);
	int total = ceil((subtotal / tax_percent) + subtotal);

	return total;
}

int main()
{
	int num_colors;
	std::cin >> num_colors;

	std::cout << get_total_paint_costs(num_colors);

	return 0;
}